### PR TITLE
Updated Readme and images

### PR DESCRIPTION
**Pull request by iwafula025**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Playbooks/Send-IngestionCostAlert/readme.md [hunk 1]
```diff
@@ -38,129 +41,197 @@ Managing cost for cloud services is an essential part of ensuring that you get m
...
 
-   ![7-computation](../Send-IngestionCostAlert/images/07-computation.png)
+   ![34-computation](../Send-IngestionCostAlert/images/34-computation.png)
 
-### In the final step below sends out an e-mail to the specified recipient list and a message is posted in a Microsoft Teams channel of your choice
+In the final step below sends out an e-mail to the specified recipient list and a message is posted in a Microsoft Teams channel that you selected earlier
 
-  ![8-sendmessage](../Send-IngestionCostAlert/images/08-sendmessage.png)
 
-  <em>Additional information on cost management can be found in this document: [https://docs.microsoft.com/en-us/azure/azure-monitor/platform/manage-cost-storage]</em>
```
Comment (sarah-yo) [inline Playbooks/Send-IngestionCostAlert/readme.md:163]: remove "en-us"

File: Playbooks/Send-IngestionCostAlert/readme.md [hunk 2]
```diff
@@ -38,129 +41,197 @@ Managing cost for cloud services is an essential part of ensuring that you get m
...
+  ![35-sendmessage](../Send-IngestionCostAlert/images/35-sendmessage.png)
+
+  <em>Additional information on cost management can be found in [this document](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/manage-cost-storage)</em>
```
Comment (sarah-yo) [inline Playbooks/Send-IngestionCostAlert/readme.md:232]: remove "en-us"

**Comment by iwafula025** [comment]
Fixed localization

File: Playbooks/Send-IngestionCostAlert/readme.md (hunk 2 above)
Comment (iwafula025) [reply Playbooks/Send-IngestionCostAlert/readme.md]: Done

**Pull request merged**